src/ext/peek-source.lisp

```diff
@@ -43,7 +43,12 @@
 
 (define-key *peek-source-keymap* "Return" 'peek-source-select)
 (define-key *peek-source-keymap* 'next-line 'peek-source-next)
+(define-key *peek-source-keymap* "C-n" 'peek-source-next)
 (define-key *peek-source-keymap* 'previous-line 'peek-source-previous)
+(define-key *peek-source-keymap* "C-p" 'peek-source-previous)
+(define-key *peek-source-keymap* "Escape" 'peek-source-quit)  ;; also C-x 0 by default.
+(define-key *peek-source-keymap* "C-c C-k" 'peek-source-quit)
```

Comment:
Well, this key binding may be good.

I came up with another one called `M-q`, what do you think?
I would like to use it in a unified operation in other places.

Reply:
To be honest, I added C-c C-k in the hope to ease the transition for Emacsers. I didn't document it in the help.

M-q is simple and good, I added it.